pkgs/development/libraries/gsettings-desktop-schemas/default.nix

```diff
@@ -35,6 +35,9 @@ stdenv.mkDerivation rec {
   postPatch = ''
     chmod +x build-aux/meson/post-install.py
     patchShebangs build-aux/meson/post-install.py
+
+    substituteInPlace meson.build \
+      --replace "join_paths(datadir, 'pkgconfig')," "join_paths(libdir, 'pkgconfig'),"
```

Comment:
This does not sound right. `pkg-config` should find them in datadir too: https://dev.gentoo.org/~mgorny/pkg-config-spec.html#package-search